Assets/LeapMotion/Modules/InteractionEngine/Scripts/InteractionBehaviour.cs

```diff
@@ -829,6 +832,9 @@ public enum GraspedMovementType {
 
     public Pose worldDeltaPose {
       get {
+        if (!_recordLastPose) _recordLastPose = true;
```

Comment:
Hmm, I don't think I agree with this optimization being so automatic.  This means that the first time the user requests the delta pose, it will be incorrect, since it was not being tracked.  And if the user has the misfortune of requesting the delta pose once every 21 frames, they will _never_ get anything but the identity pose.

If this is actually something that we want to optimize, I would take a more transparent approach and have worldDeltaPose simply throw an error if you access it without first enabling a flag manually.  That being said, is this something that is a significant performance hit?  It seems like the only work that is being bypassed is the creation and assignment of a pose struct, which should be very very fast (probably faster even than the profiler samples that wrap those operations).

Reply:
Could have been that the profiler samples were causing most of the overhead, but this feature isn't used _anywhere_ for any purpose right now -- and may wind up changing again entirely as `Pose` itself matures. Adding an unused measurement per-object is an anti-optimization even if it's relatively cheap, so I'm going to go with the script-only-flag strategy.

Reply:
(actually, just going to delete it)